proposals/blogProposal.md

```diff
@@ -0,0 +1,5 @@
+The Web of Things standardization activities are now visible(?) in our new animation video that we have been working on for the last couple of months. You can see how any web developer can implement Internet of Things solutions while being interoperable and not locked-in to framework. Additionally, the W3C WoT allows the experience of the developers to be transferred to different application domains such as agriculture, building or industrial automation, smart home or more!
```

Comment:
"locked-in to framework" --> "locked intoframeworks" ?
"smart home or more!" --> "smart home and more!" ?

Reply:
Done mostly. Just wasn't sure about the word visible since nothing was not visible before. I wanted to use something like "can be better understood" but that was also not super positive :(